sdk/src/main/java/com/virgilsecurity/sdk/cards/CardManager.java

```diff
@@ -973,6 +1030,77 @@ private Card publishRawSignedModel(RawSignedModel cardModel, TokenContext tokenC
     return result;
   }
 
+  private Card deleteRawSignedModel(RawSignedModel cardModel,
```

Comment:
What's the point to return Card here? The only thing we need is to check signatures. Card without public key is useless.

Reply:
Also returning Card without public key from CardManager on get is also pointless. All we need is probably check signatures and point that card is deleted.